tests/Unit/DataStructures/Tensor/Expressions/Test_TensorExpressions.cpp

```diff
@@ -15,6 +15,15 @@
 
 SPECTRE_TEST_CASE("Unit.DataStructures.Tensor.Expression.AddSubtract",
                   "[DataStructures][Unit]") {
+  // Test adding scalars
+  Tensor<double> scalar_1{};
```

Comment:
Can you do `scalar_1{{{2.1}}}` and `scalar_2{{{-0.8}}}` to construct in place? You should then also be able to make them `const`

Reply:
Done!